README.md

```diff
@@ -3,30 +3,46 @@
 ## About this Repository
 
 This repo is used to store the Dockerfile which can be used to build an
-Elasticsearch Docker image. The Dockerfile was generated from the [Elasticsearch
-repository](https://github.com/elastic/elasticsearch). Please note that **issues
-are disabled on this repo** and that all issues and PRs must be filed in the
-[Elasticsearch repository](https://github.com/elastic/elasticsearch).
+Elasticsearch and a Kibana Docker image. The Dockerfile was generated from their 
```

Comment:
Thoughts on generalizing for the Elastic stack? I suspect all other projects to follow suit.

Reply:
I thought about that @tylersmalley . I didn't get in a consensus with myself ( 😄) however maybe the best way would be just do what I did for Kibana, extending the related information for each stack product. Maybe for that paragraph we can just use the following:

```
This repo is used to store the Dockerfile which can be used to build Docker images for each product released in the stack. Those Dockerfiles were generated from the products' own repositories for which you can get the links in the sections below. Please note that **issues
are disabled on this repo** and that all issues and PRs must be filed in the products' repositories.
``` 

What do u think?

Reply:
I like the generic paragraph and separate sections for each project idea.